x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/DataFrameMessages.java

```diff
@@ -14,6 +14,19 @@
     public static final String REST_STOP_JOB_WAIT_FOR_COMPLETION_TIMEOUT =
             "Timed out after [{0}] while waiting for data frame job [{1}] to stop";
     public static final String REST_STOP_JOB_WAIT_FOR_COMPLETION_INTERRUPT = "Interrupted while waiting for data frame job [{0}] to stop";
+    public static final String REST_PUT_DATA_FRAME_JOB_EXISTS = "Job with id [{1}] already exists";
+    public static final String REST_DELETE_DATA_FRAME_UNKNOWN_JOB = "Job with id [{1}] could not be found";
```

Comment:
These two formatted strings should contain `{0}`
```suggestion
    public static final String REST_DELETE_DATA_FRAME_UNKNOWN_JOB = "Job with id [{0}] could not be found";
```